modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -18552,6 +18553,11 @@ user-name-and-image=User Name and Image
 user-name-field=User Name Field
 user-not-found-you-could-invite-a-new-user=User not found. You can invite a new user by typing his/her email address.
 user-notification=User Notification
+user-notification-configuration-name=User Notification
+user-notification-event-check-interval=User Notification Event Check Interval
+user-notification-event-check-interval-key-description=Set the time interval (in days) to run a scheduled job that deletes all archived user notification events older than the given number of days. Set to -1 to disable.
```

Comment:
The wording here is a bit confusing. It seems to imply that notifications older than this interval will be deleted, but it actually refers to the day limit property. Can you reword this so that it is more clear what number of days is it referring to?

Reply:
hi @adolfopa I have clarified the wording.